go/engine/identify2_test.go

```diff
@@ -71,11 +71,11 @@ func (i *Identify2WithUIDTester) MakeProofChecker(_ libkb.RemoteProofChainLink)
 	return i, nil
 }
 
-func (i *Identify2WithUIDTester) CheckHint(h libkb.SigHint) libkb.ProofError {
+func (i *Identify2WithUIDTester) CheckHint(_ *libkb.GlobalContext, h libkb.SigHint) libkb.ProofError {
```

Comment:
The other alternative is to ferret the global context away in the proof scraper classes, but we'll have to do a whole lot of plumbing there too, and it's slightly less convenient.